yelp_package/dockerfiles/itest/fake_sensu/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2015 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:trusty
+RUN apt-get update && apt-get install -yq socat
+CMD socat tcp-listen:3030,fork open:/dev/null,ignoreeof
```

Comment:
While I understand why you did this, it does add yet another docker image to build and orchistrate.

What if the sensu_host could be None and then we could just skip this under test?
And it would allow people to deploy paasta incrementally, without sensu.